lib/game/game_manager.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Reactor.GameManager do
+  use GenServer
+  @name __MODULE__
+
+  ##client API
+
+  def start_link do
+    GenServer.start_link(@name, :ok, name: @name)
+  end
+
+  def create_game do
+    Reactor.Random.random_string(10)
+    |> create_game
+  end
+
+  defp create_game(id) do
+    GenServer.call(@name, {:add_game, id})
+  end
+
+  def get_games do
+    GenServer.call(@name, {:get_games})
+  end
+
+  ##server callbacks
+
+  def init(:ok) do
+    state = %{games: %{}}
```

Comment:
It seems like the `games:` key isn't necessary here. The entire state map represents games so it seems odd to have an individual key for a single nested map.

Were you planning on adding another key to hold more state?